### ISSUE TITLE
Make content blocks not appear as cards

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#9293]
Tied to work needed in a redesign mentioned in https://github.com/rancher/dashboard/issues/9090, we have a few objects that look like cards that shouldn't be.

We need to restyle the current Card component (with class `.card-container`).

Currently, it is used in the following components:

![image](https://github.com/rancher/dashboard/assets/120640736/9e21b0d2-bfaf-4b32-b119-b204525aaaa7)

In most of them, the `box-shadow` is removed.

The idea is to turn this component into a "content-block" or "panel" component without shadow or borders. If we ever need a panel to have borders, we can include an option.

Please check the [Cards redesign Confluence page](https://confluence.suse.com/display/CU/Cards+redesign), especially the first section "Unnecessary cards" to see how those content blocks work without any border (or shadow).

@kwwii can offer some direction since this could be easy based on CSS but it depends on the components. Please document all changes for the design reviewer to check and adjust if necessary. 

For Q4, the goal would be to then take what are actual cards and update their appearance.

**Comment by sahilmulla** [comment]
Hi! I would like to make the change.

**Comment by kwwii** [comment]
@sahilmulla yes, some things are currently cards, but should not be. Some other things look like cards (they have a border) but should not.

**Comment by sahilmulla** [comment]
@kwwii are we planning to use this component as a card with borders and without borders as per needs?
